src/site/content/en/react/performance-as-a-default-with-nextjs/index.md

```diff
@@ -0,0 +1,152 @@
+by providing a complete build toolchain with a single command.
+
+Although there are a few default optimizations baked into CRA, the tool aims to provide a
+simple and straightforward setup. The choice is given to developers to decide whether
+to [eject](https://create-react-app.dev/docs/available-scripts#npm-run-eject) and
+modify the configurations themselves.
+
+Next.js, which can also be used to create a new React application, takes a different approach.
+It immediately provides a number of common optimizations that many developers would like to have
+but find difficult to set up, such as:
+
+* Server-side rendering
+* Automatic code-splitting
+* Route prefetching
+* File-system routing
+* CSS-in-JS styling (`styled-jsx`)
+
+## Setting up
+
+To create a new Next.js application, run the following command:
+
+```bash
+npx create-next-app new-app
+```
+
+{% Aside %}
+  [npx](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)
+  is a package runner that is installed automatically with npm 5.2.0 or later. It simplifies
+  quite a few processes involved with managing packages including running CLI commands
+  (like `create-next-app`) without having to install them globally on your machine.
+{% endAside %}
+
+Then navigate to the directory and start the development server:
+
+```bash
+cd new-app
+npm run dev
+```
+
+{% Aside %}
+  You can also add Next.js to an existing React application.
+  Check out [Manual Setup](https://nextjs.org/docs#manual-setup) to learn how.
+{% endAside %}
+
+The following embed shows the directory structure of a new Next.js app. Click **Remix to
+Edit** to play around with the code, and click **View App** to see the live page. The page
+will reload automatically after every edit you make.
+
+<iframe src="https://glitch.com/embed/#!/embed/new-next-app?path=index.html&attributionHidden=true" 
+        alt="A basic Next.js app on Glitch" 
+        style="height: 100%; width: 100%; border: 0;"></iframe>
+
+Notice that a `pages/` directory is created with a single file: `index.js`. Next.js follows
+a file-system routing approach, where every page within this directory is served as a separate
+route. Creating a new file in this directory, such as `about.js`, will automatically create a
+new route (`/about`).
+
+Components can also be created and used like any other React application. A `components/`
+directory has already been created with a single component, `nav.js`, which is already
+imported in `index.js`. By default, every import used in Next.js is only fetched when that
+page is loaded, providing the benefits of **automated code-splitting**.
+
+Moreover, every initial page load in Next.js is **server-side rendered**. If you open
+the Network panel in DevTools, you can see the initial request for the document returns
+a fully server-rendered page.
+
+<figure class="w-figure">
+  <img src="devtools.png" 
+       alt="The Preview tab of the Network panel shows that Next.js returns
```

Comment:
Technically alt text and figcaption text shouldn't duplicate one another. Since they'll both be announced by assistive tech, it creates a double announcement.

The relationship between alt text and figure captions is kind of dependent on the context in which the image is used. The analogy I like to think of is to imagine you're in an art gallery looking at a painting. The alt text would describe the painting scene and the caption would be the little piece of paper next to the painting that says things like the title of the painting, and maybe talks about what the artist was trying to symbolically convey.

In this instance it's tricky because the caption is describing the image so it may not be the worst thing for them to essentially duplicate, but I wanted to flag it for future images.

Btw this article by web aim is an interesting read. They show how the same painting of George Washington can have different alt text and captions and how the two relate to one another.
https://webaim.org/techniques/alttext/

Reply:
Thanks for the explanation. I created #1689 to document this difference.